saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -285,28 +282,23 @@ def _refresh_lines_unit_price(cls, info, instance, cleaned_input, new_instance):
         if new_instance:
             # It is a new instance, all new lines have already updated prices.
             return
+
         shipping_address = cleaned_input.get("shipping_address")
         if shipping_address and instance.is_shipping_required():
-            update_order_prices(
-                instance,
-                info.context.plugins,
-                info.context.site.settings.include_taxes_in_prices,
-            )
+            cls._invalidate_prices = True
+
         billing_address = cleaned_input.get("billing_address")
         if billing_address and not instance.is_shipping_required():
-            update_order_prices(
-                instance,
-                info.context.plugins,
-                info.context.site.settings.include_taxes_in_prices,
-            )
+            cls._invalidate_prices = True
 
     @classmethod
     def invalidate_prices(cls, instance, cleaned_input) -> bool:
-        return False
+        return cls._invalidate_prices
 
     @classmethod
     @traced_atomic_transaction()
     def save(cls, info, instance, cleaned_input):
+        cls._invalidate_prices = False
```

Comment:
line - 337 `recalculate_order` calculates prices. Why do we need it? It will be fully replaced when someone asks about any order price, right?